docs/bulk_data.rst

```diff
@@ -164,7 +203,9 @@ If no such mapping is provided, CumulusCI will remove the Salesforce Id from ext
 data and replace it with an autoincrementing integer primary key.
 
 Use of integer primary keys may help yield more readable text diffs when storing data in SQL
-script format. However, it comes at some performance penalty when extracting data.
+script format. However, it comes at some performance penalty when extracting data. It's
+recommended that most mappings do not map the Id field and allow CumulusCI to utilize
+the automatic primary key.
 
 Handling Namespaces
```

Comment:
Is this out of date now? The goal of the PR is to avoid having to maintain two mapping files, right?

Reply:
Yes, good point - this will need to be rewritten, as will the mapping generator.